### PR TITLE
New Event: Devoted Educated Slave

### PR DESCRIPTION
**Pull request by freecitiesbandit**
New event, firing from a devoted, intelligent and educated slave. Additional options available for paternalist and degradationist arcologies.

### CODE REVIEW

File: src/uncategorized/RESSTR.tw [hunk 1]
```diff
@@ -124,6 +135,55 @@ She does her very best to get through the inspection without disgrace, hugging h
...
+	<<set $rep += 100>>
+	<</replace>>
+<</link>>
+<<elseif $arcologies[0].FSPaternalist !== "unset">>
+<br><<link "Allow her to audit some advanced classes at Paternalist slave schools">>
+	<<replace "#name">>$activeSlave.slaveName<</replace>>
+	<<replace "#result">>
+	You tell her to take the day off for a trip to audit classes at $arcologies[0].name's most renowned Paternalist slave schools. $activeSlave.slaveName is shocked into a reverie by your words before running to wrap her arms around you in a tight hug. When she eventually untangles herself from you, she races off to get ready for her day of intellectualism while you direct $assistantName to make the necessary arrangements.
+	<br><br>
+	When $activeSlave.slaveName returns, she looks exhausted but happier than you've ever seen her. It seems she benefited greatly from $arcologies[0].name's Paternalist institutions, and has grown to @@.mediumaquamarine;trust you more@@ while also @@.hotpink;submitting to her new life as a slave.@@
```
Comment (fcpmanon) [inline src/uncategorized/RESSTR.tw]: @@.hotpink;submitting to her new life as a slave.@@

to

@@.hotpink;deepening her acceptance of slavery.@@

File: src/uncategorized/RESSTR.tw [hunk 2]
```diff
@@ -124,6 +135,55 @@ She does her very best to get through the inspection without disgrace, hugging h
...
+	<br><br>
+	When you return later in the evening, it becomes abundantly clear that your citizenry taught $activeSlave.slaveName a harsh lesson about a slave's place in $arcologies[0].name. $activeSlave.slaveName has certainly @@.green;learned to keep any pretentious thoughts about her education in her head@@. She did, however, @@.red;have quite a rough time@@ sucking all those dicks<<if $activeSlave.oralSkill <= 30>>, though she did learn about sucking dick, so she can't claim enslavement isn't educational.<<set $skillIncrease = 10>><<OralSkillIncrease $activeSlave>><<else>>.<</if>> And last of all, you and $activeSlave.slaveName did make @@.green;quite a good impression@@ today, though for widely differing reasons.
+	<<set $activeSlave.health -= 10>>
+	<<set $activeSlave.sexualFlaw = "none">>
```
Comment (fcpmanon) [inline src/uncategorized/RESSTR.tw]: why clearing sexualFlaw here?

File: src/uncategorized/RESSTR.tw [hunk 3]
```diff
@@ -124,6 +135,55 @@ She does her very best to get through the inspection without disgrace, hugging h
...
+	<<set $seed = random(65,80)>>
+	<<set $activeSlave.oralCount += $seed>>
+	<<set $oralTotal += $seed>>
+	<<set $rep += 100>>
+	<</replace>>
+<</link>>
+<<elseif $arcologies[0].FSPaternalist !== "unset">>
+<br><<link "Allow her to audit some advanced classes at Paternalist slave schools">>
```
Comment (fcpmanon) [inline src/uncategorized/RESSTR.tw]: since this option is statswise equivalent to a combination of the first two, it should have a cost added

**Comment by fcpmanon** [comment]
there is a vanilla typo - distangling instead fo disentangling

File: src/uncategorized/RESSTR.tw (hunk 3 above)
Comment (freecitiesbandit) [inline src/uncategorized/RESSTR.tw]: My place on it is that it has an additional cost already - being a paternalist arcology. Slight barrier to entry for a slightly better reward

Comment (freecitiesdev) [inline src/uncategorized/RESSTR.tw]: That's fine, and in fact I'd support that kind of thing going forward. FS locked choices should be somewhat better.

**Comment by freecitiesdev** [comment]
In randomIndividualEvent.tw, you have this:

<<if ($eventSlave.intelligence > 1)>>
 <<if ($eventSlave.devotion > 50)>>
 <<if ($eventSlave.intelligenceImplant > 0)>>
 	<<set $RESSTRevent.push("devoted educated slave")>>

Which needed to be:

<<if ($eventSlave.intelligence > 1)>>
 <<if ($eventSlave.devotion > 50)>>
 <<if ($eventSlave.intelligenceImplant > 0)>>
 	<<set $RESSTRevent.push("devoted educated slave")>>
<</if>>
<</if>>
<</if>>

Every <<if>> needs a closing <</if>>. Pretty catastrophic.

**Comment by freecitiesdev** [comment]
Minor event writing issues:

- <<Master>> is for slave speech, not the PA. She uses sir/ma'am.

- Take care with punctuation around quotations. I've made some edits there.

**Pull request merged**